server/main.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func main() {
-	conf, err := config.FromOsEnv()
+	conf, err := config.FromEnvFile()
```

Comment:
Read environment file (`.env`) instead of OS environment variables

Reply:
We need to read both of them

Reply:
I think we need to define a way to gather environment vsriables either by reading a `.env` file or OS environment variables.

What do you think about working om this on another PR?